<?xml version="1.0"?>
<launch>
  <param name="scale_linear" value="2" type="double"/>
    <param name="scale_angular" value="2" type="double"/>

    <node pkg="beginner_tutorials" type="talker"
          args="/talk" name="talker" />
  <test pkg="beginner_tutorials" test-name="tf_talker" type="tf_talker" />
</launch>
